src/encoded/schemas/changelogs/target.md

```diff
@@ -3,6 +3,10 @@
 ### Schema version 8
 
 * Move status property to standard_status mixin.
+* Added "targeted_genes" property for the list of genes targeted by assay or antibody associated with the target.
```

Comment:
These changes don't require an upgrade as they're new properties, so perhaps they should be listed as minor changes since schema version 7... or 8.

Reply:
What is the correct way/format to list the changes as minor changes? Target is version 8 in production now.